x/slashing/internal/keeper/infractions.go

```diff
@@ -143,9 +112,9 @@ func (k Keeper) HandleDoubleSign(ctx sdk.Context, addr crypto.Address, infractio
 	k.SetValidatorSigningInfo(ctx, consAddr, signInfo)
 }
 
-// handle a validator signature, must be called once per validator per block
-// TODO refactor to take in a consensus address, additionally should maybe just take in the pubkey too
+// HandleValidatorSignature handles a validator signature, must be called once per validator per block.
 func (k Keeper) HandleValidatorSignature(ctx sdk.Context, addr crypto.Address, power int64, signed bool) {
+	// TODO: refactor to take in a consensus address, additionally should maybe just take in the pubkey too
```

Comment:
should an issue be opened for this TODO?

Reply:
I removed the TODO as the validator's consensus votes are stored with the `crypto.Address` and it'd require to move these changes (consAddr and pubkey) to the `EndBlock`, which I don't think is necessary. 

Also, HandleDoubleSign does it like this and doesn't have the TODO.